checkstyle/suppressions.xml

```diff
@@ -540,6 +540,8 @@
     <suppress checks="ClassFanOutComplexityCheck" files="com.hazelcast.map.impl.proxy.MapProxySupport"/>
     <suppress checks="ClassDataAbstractionCoupling" files="com.hazelcast.map.impl.proxy.MapProxySupport"/>
     <suppress checks="CyclomaticComplexity" files="com.hazelcast.map.impl.client.AbstractTxnMapRequest"/>
+    <suppress checks="MethodCount|ClassDataAbstractionCoupling" files="com.hazelcast.map.impl.MapServiceContextImpl"/>
+    <suppress checks="MethodCount" files="com.hazelcast.map.impl.MapServiceContext"/>
```

Comment:
I guess what's confusing is that the title says simplifications but then there's checkstyle suppressions